README.md

```diff
@@ -1,67 +1,9 @@
-![cf](https://i.imgur.com/7v5ASc8.png) Lab 11: Express and Mongo REST API
-======
+npm run dbon to start mongo database
```

Comment:
Add more information about what the application is and how the server responds to valid/invalid requests.